recipes/ua-parser/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{%set name = "ua-parser" %}
+{%set version = "0.7.1" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "e1f7cb16ccb68d6c93b19348e678500bec8cd1f692e49214dd38ca2d100ada9b" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pyyaml
```

Comment:
Ack! 😱 Sorry, @pmlandwehr, must have missed that issue for some reason. Let's leave it alone after all.